pkgs/by-name/ap/apple-sdk/package.nix

```diff
@@ -37,8 +38,10 @@ let
       (callPackage ./common/remove-disallowed-packages.nix { })
     ]
     # Only process stubs and convert them to tbd-v4 if jq is available. This can be made unconditional once
-    # the bootstrap tools have jq and libtapi.
-    ++ lib.optional (jq != null) (callPackage ./common/process-stubs.nix { })
+    # the bootstrap tools have jq and llvm-readtapi.
```

Comment:
This is already trivially true because they have LLVM or this will be true after a bootstrap tools bump?

Reply:
The bootstrap tools need to include `llvm-readtapi`, which they don’t. I have a separate PR I will be opening for that.

Reply:
Should this be draft then? Otherwise bootstrapping will be broken on staging, no?

Reply:
No, it’ll work as before because of the `lib.optional (jq != null && lib.getName llvm != "bootstrap-stage0-llvm")` conditional.